src/mumble/ACLEditor.cpp

```diff
@@ -827,18 +827,26 @@ void ACLEditor::on_qcbACLInherit_clicked(bool) {
 
 void ACLEditor::on_qcbACLApplyHere_clicked(bool checked) {
 	ChanACL *as = currentACL();
-	if (!as || as->bInherited)
+	if (!as || as->bInherited) {
 		return;
+	}
 
 	as->bApplyHere = checked;
+	if (!checked && !qcbACLApplySubs->isChecked()) {
+		qcbACLApplySubs->setCheckState(Qt::Checked);
+	}
```

Comment:
For this kind of behavior a RadioButton is likely the better UI element as this is more intuitively associated with having only a single state at a time.
Alternatively, a combobox might also work :thinking:

Reply:
But it can be either one or both :D
Maybe turn it into a radio button and add the ``both`` option?

Reply:
Ah okay - then I misunderstood the code here. Nevermind then :D
(RadioBox with Both is a possibility but I think two checkbox es are prettier in this case)